### ISSUE TITLE
Handle component multi-word with Nuxt components prefix

### ISSUE THREAD

**Issue by Atinux** [nuxt/eslint#261]
We should not have those warning with Nuxt since we do prefix by default:
![CleanShot 2023-01-10 at 17.04.15@2x.png](https://volta.s3.fr-par.scw.cloud/Clean_Shot_2023_01_10_at_17_04_15_2x_391169efa4.png)

You can test easily on https://github.com/nuxtlabs/nuxtlabs.com/tree/eslint-config @danielroe with running `npm run lint`

**Comment by MuhammadM1998** [comment]
I stumbled on this today. I'm setting a custom prefix in `nuxt.config`
```ts
export default defineNuxtConfig({
  modules: ['@nuxt/eslint'],
  components: [{ path: '/components', prefix: 'CB' }],
})
```

With the following file tree
```asciidoc
| components
  | --- Logo.vue
```

I get the following error
```shell
D:\my-base-layer\components\Logo.vue
  1:1  error  Component name "Logo" should always be multi-word  vue/multi-word-component-names
```

Removing the custom prefix from the config and moving all the components to `components/CB` folder yields the same error

**Comment by antfu** [comment]
The issue should be solved with the 0.3 rewrite and no longer relevant.

@MuhammadM1998 Please create another issue with minimal reproduction. Thanks.

**Comment by MuhammadM1998** [comment]
@antfu I've created a reproduction and an issue https://github.com/nuxt/eslint/issues/419
